src/map/lua/lua_baseentity.cpp

```diff
@@ -10236,6 +10236,98 @@ void CLuaBaseEntity::recalculateAbilitiesTable()
     PChar->pushPacket(new CCharAbilitiesPacket(PChar));
 }
 
+/************************************************************************
+ *  Function: getEntitiesInRange()
+ *  Purpose : Returns a Lua table of entities within range of the base entity
+ *  Example : local players = npc:getEntitiesInRange(target, aoeType, radiusOrigin, distance, findFlags, validTargets)
+ *  Notes   : if the passed distance is nil or 0 (as well as many other combinations), empty table will be returned
```

Comment:
I would add a _big fat note_ here, saying something to the tune of: "APART FROM THIS ONE EXCEPTION, ALL TARGET AND PATH FINDING SHOULD BE DONE EXCLUSIVELY IN CORE"

Reply:
and say that this is used for Ixion, and possibly there's another usage for this in assault? I don't remember

Reply:
Can do

Reply:
And another sub-note: If you have to do expensive operations that are meant for core in Lua, you should be doing them as infrequently as possible: In a mobskill that locks itself off, or anything that isn't on-tick, etc.

Reply:
Updated the description